### PR TITLE
added HttpResponseRedirect to views.py and assigned in to oscnRedirec…

### PR DESCRIPTION
**Pull request by Serhazor**
…t function, getting a different type of error, please check if I've done it correctly

### CODE REVIEW

File: website/templates/index.html [hunk 1]
```diff
@@ -110,7 +110,11 @@ <h4>1. How do I know my OSCN Case Number?</h4>
 			</div>
 			<div class="col-sm-6">
 				<h4>2. What if I still can't find my case number?</h4>
-				<p>Check with your public defender or search your name online at <a href=“www.oscn.net/dockets/search.aspx”>www.oscn.net/dockets/search.aspx </a> </p> </div> <div class="col-sm-6">
+				<p>Check with your public defender or search your name online at <a href="http://{{oscnRedirect}}">www.oscn.net/dockets/search.aspx </a>
```
Comment (jwhitlock) [inline website/templates/index.html:113]: Here's how to make the URL an absolute URL:

```suggestion
				<p>Check with your public defender or search your name online at <a href="https://www.oscn.net/dockets/search.aspx">www.oscn.net/dockets/search.aspx</a> 
```

File: website/templates/index.html [hunk 2]
```diff
@@ -110,7 +110,11 @@ <h4>1. How do I know my OSCN Case Number?</h4>
 			</div>
 			<div class="col-sm-6">
 				<h4>2. What if I still can't find my case number?</h4>
-				<p>Check with your public defender or search your name online at <a href=“www.oscn.net/dockets/search.aspx”>www.oscn.net/dockets/search.aspx </a> </p> </div> <div class="col-sm-6">
+				<p>Check with your public defender or search your name online at <a href="http://{{oscnRedirect}}">www.oscn.net/dockets/search.aspx </a> 
+				</p> 
+			</div> 
+			
+			<div class="col-sm-6">
```
Comment (jwhitlock) [inline website/templates/index.html:117]: 👍 good job reformatting this section

File: website/urls.py [hunk 3]
```diff
@@ -5,3 +5,7 @@
     path('', views.index, name='index'),
     path('schedule_reminders', views.schedule_reminders, name='schedule_reminders')
 ]
+
+
+
+
```
Comment (jwhitlock) [inline website/urls.py:11]: Please remove these extra lines

File: website/views.py [hunk 4]
```diff
@@ -12,6 +12,9 @@
 
 from alerts.models import Alert
 
+def oscnRedirect(request):
```
Comment (jwhitlock) [inline website/views.py:15]: The changes to this file is not needed to fix the issue.

Comment (Serhazor) [reply website/views.py]: Done

**Pull request merged**